plugins/module_utils/httpbearerauth.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, Ansible Project
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+import requests
```

Comment:
Try importing as described here https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/import.html#import

Reply:
Well, now traceback is giving: NameError: name 'requests' is not defined. pip install requests is usually all that's needed, or yum install python-requests python36-requests.